Unknown type name 'boolean' in jpeglib.h when WITH_JPEG is on.

```
[ 47%] Building C object libfreerdp/codec/CMakeFiles/freerdp-codec.dir/jpeg.c.o
In file included from /home/floppym/src/FreeRDP/libfreerdp/codec/jpeg.c:33:0:
/usr/include/jpeglib.h:96:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:112:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:167:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:282:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:297:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:366:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:367:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:368:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:369:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:371:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:386:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:396:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:412:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:472:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:496:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:497:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:500:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:501:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:503:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:506:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:509:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:510:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:511:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:599:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:601:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:602:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:613:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:620:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:623:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:794:3: error: expected specifier-qualifier-list before ‘boolean’
/usr/include/jpeglib.h:806:3: error: expected specifier-qualifier-list before ‘boolean’
/usr/include/jpeglib.h:844:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:850:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:857:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:862:3: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:885:9: error: expected declaration specifiers or ‘...’ before ‘*’ token
/usr/include/jpeglib.h:1018:31: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1020:38: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1024:35: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1027:35: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1033:35: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1039:34: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1070:30: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1083:1: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1087:1: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1095:1: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1096:1: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1098:1: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1099:1: error: unknown type name ‘boolean’
/usr/include/jpeglib.h:1122:2: error: unknown type name ‘jpeg_marker_parser_method’
/usr/include/jpeglib.h:1148:1: error: unknown type name ‘boolean’
/home/floppym/src/FreeRDP/libfreerdp/codec/jpeg.c: In function ‘do_decompress’:
/home/floppym/src/FreeRDP/libfreerdp/codec/jpeg.c:92:9: error: ‘struct jpeg_source_mgr’ has no member named ‘fill_input_buffer’
/home/floppym/src/FreeRDP/libfreerdp/codec/jpeg.c:93:9: error: ‘struct jpeg_source_mgr’ has no member named ‘skip_input_data’
/home/floppym/src/FreeRDP/libfreerdp/codec/jpeg.c:94:9: error: ‘struct jpeg_source_mgr’ has no member named ‘resync_to_restart’
/home/floppym/src/FreeRDP/libfreerdp/codec/jpeg.c:95:9: error: ‘struct jpeg_source_mgr’ has no member named ‘term_source’
/home/floppym/src/FreeRDP/libfreerdp/codec/jpeg.c:102:2: warning: implicit declaration of function ‘jpeg_read_header’ [-Wimplicit-function-declaration]
make[2]: *** [libfreerdp/codec/CMakeFiles/freerdp-codec.dir/jpeg.c.o] Error 1
make[1]: *** [libfreerdp/codec/CMakeFiles/freerdp-codec.dir/all] Error 2
make: *** [all] Error 2
```

--- a/libfreerdp/codec/jpeg.c
+++ b/libfreerdp/codec/jpeg.c
@@ -28,7 +28,6 @@
 #ifdef WITH_JPEG
 
 #define XMD_H
-#define HAVE_BOOLEAN
 
 #include <jpeglib.h>
 

